data/flexibleForms/mapFormIdsToDisplayName.ts

```diff
@@ -1,4 +1,10 @@
 export const mapIdToType: Record<string, string> = {
   'adult-case-note': 'Case note',
   'child-case-note': 'Case note',
+  foo: 'Sandbox form',
+  'review-3c': 'Review of Care and Support Plan (3C)',
+  'FACE overview assessment': 'FACE overview assessment',
```

Comment:
These don't really feel like IDs? "FACE overview assessment" doesn't at least. Is that the intention?

Reply:
It seems more logical to me that we'd have one mapping object for this and the other use cases, something like:

```ts
type FormDefinition = {
  form: Form;
  displayName: string;
  id: string;
};

const formMap = Record<string, FormDefinition> = {
  'adult-case-note': {
    form: formAsObjects.adultCaseNote,
    displayName: "Case note",
    id: "adult-case-note",
  },
  // ...
}
```

Reply:
Agreed on the single mapping.

The "id" thing is a bit messy, as "FACE overview assessment" is not currently released there is nothing stopping me just changing its id to be hyphenated like the rest of the forms.